app/views/users/backup_code_setup/create.html.erb

```diff
@@ -57,3 +57,7 @@
       ) %>
   <%= f.submit t('forms.buttons.continue'), class: 'margin-top-2' %>
 <% end %>
+
+<%= render PageFooterComponent.new do %>
+  <%= link_to t('two_factor_authentication.backup_codes.add_another_authentication_option'), authentication_methods_setup_path %>
```

Comment:
True. Another problem with that is it lets you move along without clicking the confirmation checkbox. It takes you back to the Select MFA page. Backup Codes are enabled at this point and if you haven't taken steps to secure them you could find yourself needing to reauth and in a bit of a pickle.
Should this go back to UX? 😟

Reply:
>Should this go back to UX? 😟

I don't think we need to go back to the drawing board, maybe just check-in and see if the actual behavior is acceptable enough. If necessary, we could remove the link, change the text to be more accurate, or maybe even make it so that clicking it will destroy the backup codes.

Reply:
Related Slack discussion: https://gsa-tts.slack.com/archives/C01710KMYUB/p1708035169458219